index.js

```diff
@@ -52,6 +52,11 @@ GulpRunner.prototype.run = function(tasks, options, cb) {
       cb(null)
     }
   });
+
+  process.on('SIGINT', function() {
+    gulp.kill();
+    process.exit(1);
```

Comment:
What situation would this fire in? I just want to make sure `1` is the right exit code.